### PR TITLE
Update Razor to 6.0.0-alpha.1.21201.19

### PR DESCRIPTION
**Pull request by NTaylorMullen**
- For this to work it requires the latest omnisharp-roslyn release.
- Aligns with the .NET 6.0 Preview3 runtime release & 16.10-Preview2 VS tooling release.

Wasn't able to fullly try this out because it looks like O#'s "latest" release blob storage is down?
image

Fixes dotnet/aspnetcore#30834

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -331,8 +331,7 @@
     {
       "id": "Razor",
       "description": "Razor Language Server (Windows / x64)",
-      "url": "https://download.visualstudio.microsoft.com/download/pr/a1999a11-02b4-49cf-9716-bdf7fb656b0b/c2388ccf719057b1b8d29aab27173206/razorlanguageserver-win-x64-6.0.0-alpha.1.20575.5.zip",
-      "fallbackUrl": "https://razorvscodetest.blob.core.windows.net/languageserver/RazorLanguageServer-win-x64-6.0.0-alpha.1.20575.5.zip",
```
Comment (NTaylorMullen) [inline package.json:335]: Separately working out where our own blob storage went. For now removing fallback url as it's not needed

File: src/razor/syntaxes/aspnetcorerazor.tmLanguage.json [hunk 2]
```diff
@@ -1524,4 +1524,4 @@
       }
     }
   }
-}
+}
```
Comment (TanayParikh) [inline src/razor/syntaxes/aspnetcorerazor.tmLanguage.json:1527]:
```suggestion
}

```

File: package.json [hunk 3]
```diff
@@ -540,17 +536,17 @@
                   "items": {
                     "type": "string"
                   },
-                  "description": "Array of symbol server URLs (example: http\u200b://MyExampleSymbolServer) or directories (example: /build/symbols) to search for .pdb files. These directories will be searched in addition to the default locations -- next to the module and the path where the pdb was originally dropped to.",
+                  "description": "Array of symbol server URLs (example: http​://MyExampleSymbolServer) or directories (example: /build/symbols) to search for .pdb files. These directories will be searched in addition to the default locations -- next to the module and the path where the pdb was originally dropped to.",
```
Comment (TanayParikh) [inline package.json:539]: Were these changes intentional?

**Review by TanayParikh — APPROVED** [review]

File: package.json (hunk 3 above)
Comment (NTaylorMullen) [reply package.json:539]: Definitely not

**Comment by NTaylorMullen** [comment]
@captainsafia with this I get spammed for Blazor debugging prompts. I assume not expected?
image

**Comment by captainsafia** [comment]
@NTaylorMullen Nope, not expected. The code that displays that prompt lives in the omnisharp-vscode repo so it's unrelated to the insertion.

Can you open an issue with more details and I'll take a look?

I don't think this should block the insertion given where the codepath lies as mentioned above.

**Comment by NTaylorMullen** [comment]
Filed: https://github.com/dotnet/aspnetcore/issues/31465

File: package-lock.json [hunk 4]
```diff
@@ -32,7 +32,7 @@
         "yauzl": "2.10.0"
       },
       "devDependencies": {
-        "@istanbuljs/nyc-config-typescript": "^1.0.1",
+        "@istanbuljs/nyc-config-typescript": "1.0.1",
```
Comment (NTaylorMullen) [inline package-lock.json:35]: I updated to latest node and ran restore. Want me to keep all of this or revert?

File: package.json [hunk 5]
```diff
@@ -1942,17 +1938,17 @@
...
+                    "description": "Array of symbol server URLs (example: http​://MyExampleSymbolServer) or directories (example: /build/symbols) to search for .pdb files. These directories will be searched in addition to the default locations -- next to the module and the path where the pdb was originally dropped to.",
                     "default": []
                   },
                   "searchMicrosoftSymbolServer": {
                     "type": "boolean",
-                    "description": "If 'true' the Microsoft Symbol server (https\u200b://msdl.microsoft.com\u200b/download/symbols) is added to the symbols search path. If unspecified, this option defaults to 'false'.",
+                    "description": "If 'true' the Microsoft Symbol server (https​://msdl.microsoft.com​/download/symbols) is added to the symbols search path. If unspecified, this option defaults to 'false'.",
```
Comment (NTaylorMullen) [inline package.json:1946]: Revert all this

File: package-lock.json (hunk 4 above)
Comment (JoeRobich) [reply package-lock.json:35]: Keep it. Better to not let versions drift between dev environments.

**Review by JoeRobich — APPROVED** [review]
